front-end/src/IndividualShow.js

```diff
@@ -0,0 +1,168 @@
+        <label for="hulu">Hulu  </label>
+        <input type="checkbox" id="apple" value="Apple" ref={refApple} />
+        <label for="apple">Apple TV+  </label><br /><br />
+        <input type="submit" value="Save Platform" />
+      </form>
+    </div>
+  )
+}
+
+/*the component displays and stores general info about the show*/
+
+const Description = ({ genre, description, total_episodes, is_movie }) => {
+  const refDescription = useRef();
+  const refIsMovie = useRef();
+  const refGenre = useRef();
+  const refTotalEpisodes = useRef();
+
+  const saveMovieData = (() => {
+    //the function will be further developed in sprint 2
+  });
+  return (
+    <div class="description">
+      <br /><br />
+      <label for="genre">Genre:  </label>
+      <input type="text" id="genre" value={genre} ref={refGenre} readonly /><br /><br />
+      <label for="description">Description: </label>
+      <textarea id="description" value={description} ref={refDescription} readonly /><br /><br />
+      <label for="total_episodes">Total Episodes: </label>
+      <input type="text" id="total_episodes" value={total_episodes} ref={refTotalEpisodes} readonly /><br /><br />
+      <label for="isMovie">Is it a movie? </label>
+      <input type="text" id="isMovie" value={is_movie} ref={refIsMovie} readonly /><br /><br />
+    </div>
+  )
+}
+
+/*the component calls other components and displays comprehensive info about the show; 
+values passed as parameters to the main components are generated by the mock data API */
+
+const IndividualShow = (props) => {
+
+  const refTitle = useRef();
+  const refCover = useRef();
+  const returnToShows = (() => {
+    window.location.href = '/my-shows/'
+  });
+  const addToWatched = (() => {
+
+    window.location.href = '/my-shows/'
+  });
+
+
+  let [show, setShow] = useState([]);
+
+  useEffect(() => {
+    //temporary variable to be replaced
+    let showInfo = [];
+    let show_id = "54";
+    //axios.get(`https://my.api.mockaroo.com/shows/${show.id}.json?key=3010e030`)
+    axios.get(`https://my.api.mockaroo.com/shows/${show_id}.json?key=3010e030`)
+      .then((response) => {
+        showInfo.push(response.data);
+        console.log(showInfo);
+        console.log(show);
+        setShow(showInfo);
+      })
+  }, [])
+
+
+  return (
+    <>
+      <Hamburger pageWrapId={'page-wrap'} outerContainerId={'outer-container'} />
+      <div className="header"><h2>Show Page</h2></div>
+
+      <div class="show_content">
+        <fieldset class="main">
+          <div class="show-details">
+            <fieldset >
+              {show.map(s => ( //display name of the show
+                <h3 id="title" value={s.name} ref={refTitle}>{s.name}</h3>
+              ))}
+              <button onClick={(e) => returnToShows()}>
+                Return to Shows
+      </button>
+              <button onClick={(e) => addToWatched()}>
+                Add to In Progress or Watched Shows
```

Comment:
The user should be given an option to mark a show as in progress or watched--there shouldn't just be one button for this.

Reply:
that was on the wireframe- you said not to change anything; i can edit that though

Reply:
The wireframe is not very fleshed out, and thinking about it logically, how will the app know whether to label a show as in progress or completed if there is only one button for doing that?